feat(path): Added a fish-like trimmed path (PWD)

aemonge opened the pull request.

Commit:
feat(path): Added a fish-like trimmed path (PWD)

xvilo commented:
I would say to check if `shrink_path`-plugin has been loaded. Then use `shrink_path` instead? I have done this in my own fork

aemonge commented:
It didn't work for me, should I fork your repo ?

xvilo commented:
I can look into creating a branch with my changes. It's sometime ago

xvilo commented:
@aemonge Please see this PR https://github.com/agnoster/agnoster-zsh-theme/pull/71

Commit:
feat(trimmedPath): Added the same trimmed fn applied to path, to git freature/branche/current -> /f/b/current

apjanke commented:
If y'all are interested, I added this to [my AgnosterJ fork](https://github.com/apjanke/agnosterj-zsh-theme).

https://github.com/apjanke/agnosterj-zsh-theme/commit/e91e59480ed4572e132b9884d280a9bc7f4334a1

<img width="585" alt="Screen Shot 2020-01-03 at 4 25 00 PM" src="https://user-images.githubusercontent.com/2618447/71750158-9d9aa400-2e45-11ea-9c29-64f11710a959.png">

Commit:
fix(path): Shrinked the path for both upper and lower case

Commit:
feat(trim): Enable disable trim path, by variable

aemonge commented:
Hey.... I didn't understood the last comment. Is this feature already implemented ?

apjanke commented:
@aemonge It looks like development has stalled out on this repo. So I created a fork for doing new development in. My fork is at https://github.com/apjanke/agnosterj-zsh-theme. This feature is already implemented over on my fork.

aemonge committed:
No Host

aemonge committed:
Added conda

Conda env, thanks to:
https://gist.github.com/Kelym/782b9838ae44dd54d0a376ca82ca2f7c

aemonge committed:
Update README.md

aemonge committed:
fixed conda color

aemonge committed:
Update agnoster.zsh-theme

aemonge committed:
Update README.md

aemonge committed:
Added rbenv

aemonge committed:
Added node, ruby, python as envs

aemonge committed:
Aws

aemonge committed:
trim branch name

aemonge committed:
fix

aemonge committed:
Change de option for tiny

THis is more significant name

aemonge committed:
fix(theme): fixed duplicated lines

aemonge committed:
Merge branch 'master' of github.com:agnoster/agnoster-zsh-theme
